src/components/index/sections/TeamSection.tsx

```diff
@@ -27,14 +32,17 @@ export default function TeamSection() {
       justify="center"
       component="section"
       className={classes.container}>
-      <Typography variant="h5" component="h2" className={classes.heading}>
+      <Typography variant="h5" className={classes.heading}>
         {t('index:team-section.heading')}
       </Typography>
-      <Grid item>
-        <Typography variant="body2">{t('index:team-section.content')}</Typography>
+      <Grid container>
+        <Grid item>
+          <Typography variant="body2">{t('index:team-section.content')}</Typography>
+        </Grid>
+        <Grid item>
+          <img src={`img/discord-team-image.png`} className={classes.teamImage} />
```

Comment:
It's better to load the image with absolute route such as `/img/discord-team-image.png` as this url will break if we use the same component deeper in the route structure. Now it works because we use it on `/` but that's special case.